modules/pubmaticBidAdapter.js

```diff
@@ -1069,6 +1069,38 @@ export function prepareMetaObject(br, bid, seat) {
   }
 }
 
+/**
+ * Checks url length is not exceeding the configured length.
+ * @param {*} payload - payload data
+ * @param {*} maxLength - length of URL characters
+ * @returns boolean
+ */
+function hasUrlLengthIsNotExceeding(payload, maxLength) {
+  const encodedPayload = btoa(JSON.stringify(payload));
```

Comment:
why are we encoding the payload with base64? won't prebid encode the URL params?

Reply:
@pm-harshad-mane - Yes, prebid encodes the URL parameters. but we want to get the length of the URL before sending it to the prebid to call (which sends it to the server) so we can compare it with a provided maxLength.

Reply:
comparing the maxLength is fine.
my question is about encoding the data in base64 using btoa even-though we know that the data will be URLencoded, why do we need to encode it with base64? can we avoid using `btoa` completely?